sdk/webpubsub/Azure.Messaging.WebPubSub/src/HealthApiClient_extensions.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+#pragma warning disable AZC0007
```

Comment:
Consider adding a TODO comment with a bug number to make it easier to clean these up when we improve the analyzers.  You can tack it on to https://github.com/Azure/azure-sdk-tools/issues/127 if that's faster.